### PR TITLE
Chore/improve seo

### PR DESCRIPTION
**Pull request by allesklardy**
- Added title and meta descriptions tags to all pages. 
- Added sitemap.xml to web root #55 
- Updated Images to .webp. Reduced Home Page load by 2MB

### CODE REVIEW

**Comment by allesklardy** [comment]
Sorry for the conflicts :))

File: src/app.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	<head>
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
+		<meta name="keywords" content="Speedclimbing Statistics, Worldranking, IFSC" />
```
Comment (dorianim) [inline src/app.html:6]: Shouldn't keywords be page-specific as well?

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -10,6 +10,15 @@
 	export let data: PageData;
 </script>
 
+<svelte:head>
+	<title>Home of Speedclimbing.org</title>
+	<meta
+		name="description"
+		content="Your Page for the current and past statistics about Speedclimbing.
```
Comment (dorianim) [inline src/routes/+page.svelte:17]:
```suggestion
		content="speedclimbing.org - the platform to get information about speedclimbing for everyone - from newbies to professionals. 
```

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -10,6 +10,15 @@
 	export let data: PageData;
 </script>
 
+<svelte:head>
+	<title>Home of Speedclimbing.org</title>
+	<meta
+		name="description"
+		content="Your Page for the current and past statistics about Speedclimbing. 
+		See everything about the current world records, the latest tournaments and the performance of the athlets."
```
Comment (dorianim) [inline src/routes/+page.svelte:18]:
```suggestion
		Get to know everything about the current world records, latest competitions and performance of the athlets."
```

File: src/routes/about/+page.svelte [hunk 4]
```diff
@@ -4,6 +4,14 @@
 	import TheWall from './TheWall.svelte';
 </script>
 
+<svelte:head>
+	<title>Everything about Speedclimbing</title>
+	<meta
+		name="description"
+		content="See the rules, the wall and the sport. Learn about the qualifications, the competitions and everything about it."
```
Comment (dorianim) [inline src/routes/about/+page.svelte:11]:
```suggestion
		content="You have never heard of speedclimbing? This is the perfect place to get up to speed and learn more about the sport!"
```

File: src/routes/athlete/+page.svelte [hunk 5]
```diff
@@ -33,6 +33,14 @@
 	];
 </script>
 
+<svelte:head>
+	<title>Athlete Search</title>
+	<meta
+		name="description"
+		content="Detailed statistics of all athletes in the database. See their personal bests, competitions and more."
```
Comment (dorianim) [inline src/routes/athlete/+page.svelte:40]:
```suggestion
		content="Detailed statistics of all your favorite athletes. Find out their personal bests, competition results and more!"
```

File: src/routes/competition/+page.svelte [hunk 6]
```diff
@@ -56,6 +56,14 @@
 	}
 </script>
 
+<svelte:head>
+	<title>Competetions - Overview</title>
```
Comment (dorianim) [inline src/routes/competition/+page.svelte:60]:
```suggestion
	<title>Competitions - Overview</title>
```

File: src/routes/competition/+page.svelte [hunk 7]
```diff
@@ -56,6 +56,14 @@
...
+	<meta
+		name="description"
+		content="Everything about the last and upcoming competitions. See the results, the records and the calendar of the speedclimbing world."
```
Comment (dorianim) [inline src/routes/competition/+page.svelte:63]:
```suggestion
		content="Everything about past and upcoming competitions. Get an overview of the whole schedule of speedclimbing events!"
```

File: src/routes/stats/summary/+page.svelte [hunk 8]
```diff
@@ -13,6 +13,15 @@
 	}
 </script>
 
+<svelte:head>
+	<title>Statistics Summary</title>
+	<meta
+		name="description"
+		content="See the summary of the current season and the all time statistics of the speedclimbing world.
```
Comment (dorianim) [inline src/routes/stats/summary/+page.svelte:20]:
```suggestion
		content="The latest and most relevant speedclimbing statistics at a glance.
```

File: src/routes/stats/summary/+page.svelte [hunk 9]
```diff
@@ -13,6 +13,15 @@
...
+		View who has the most medals and the world records."
```
Comment (dorianim) [inline src/routes/stats/summary/+page.svelte:21]:
```suggestion
		Find out who has won most medals and get to know the fastest climbers in the world!"
```

**Review by dorianim — CHANGES_REQUESTED** [review]
Thanks! Looks great overall, but I's like to rephrase some of the descriptions :)

Also, I think, we should generate the sitemap dynamically instead of having a static xml file. This has the benefit, that we can generate entries for all athletes/competitions once we have pages for them.

File: src/app.html (hunk 1 above)
Comment (allesklardy) [reply src/app.html:6]: They aren't considered for Google, but i wanted to add them just in case

Comment (dorianim) [reply src/app.html:6]: Ok, I didn't know that.

**Review by dorianim — CHANGES_REQUESTED** [review]
Just one more thing:
Please add yourself to the contributers on the about us page :)

**Review by dorianim — APPROVED** [review]

**Pull request merged**